src/FakeItEasy/Configuration/IAfterCallSpecifiedConfiguration.cs

```diff
@@ -3,8 +3,9 @@ namespace FakeItEasy.Configuration
     /// <summary>
     /// Lets you set up expectations and configure repeat for the configured call.
     /// </summary>
-    public interface IAfterCallSpecifiedConfiguration
-        : IRepeatConfiguration

```

Comment:
While we're breaking things, why not?
I assume very few people refer to these types by name.